alerts/build.gradle

```diff
@@ -41,8 +40,9 @@ kotlin {
         getByName("commonMain") {
             dependencies {
                 implementation project(":base", "")
-                implementation 'co.touchlab:stately:0.9.5'
+                implementation 'co.touchlab:stately-common:1.0.0-a2'
+                implementation 'co.touchlab:stately-concurrency:1.0.0-a2'
```

Comment:
Should be a variable, as in @Daeda88 branch. It's already copied in HUD, logging etc